README.md

```diff
@@ -211,6 +211,7 @@ PcapPlusPlus currently supports parsing, editing and creation of packets of the
 9. VLAN
 10. VXLAN
 11. Wake on LAN (WoL)
+12. NFLOG (Linux Netfilter NFLOG)

```

Comment:
Since we only have parsing capabilities for now please add `parsing only (no editing capabilities)`:
```markdown
NFLOG (Linux Netfilter NFLOG) - parsing only (no editing capabilities)
```